testing/xfpga/test_sysfs_c.cpp

```diff
@@ -160,6 +161,30 @@ TEST_P(sysfsinit_c_p, sysfs_get_region) {
   EXPECT_EQ(0, sysfs_finalize());
 }
 
+/**
+* @test   get_interface_id
+* @details Given a valid token
+           When I call sysfs_get_interface_id with that token
+*          I get the expected interface_id
+*/
+TEST_P(sysfsinit_c_p, get_interface_id) {
+  fpga_guid guid;
+  fpga_properties props;
+  fpga_token fme;
+  uint32_t matches = 0;
+  fpga_guid parsed_guid;
+  ASSERT_EQ(sysfs_initialize(), 0);
+  ASSERT_EQ(fpgaGetProperties(nullptr, &props), FPGA_OK);
+  ASSERT_EQ(fpgaPropertiesSetObjectType(props, FPGA_DEVICE), FPGA_OK);
+  ASSERT_EQ(xfpga_fpgaEnumerate(&props, 1, &fme, 1, &matches), FPGA_OK);
```

Comment:
Doesn't props and fme have to be destroyed?

Reply:
Oh yeah, thanks!